spec/models/concerns/courses/edit_options/age_range_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe Courses::EditOptions::AgeRangeConcern do
+  let(:example_model) do
+    klass = Class.new do
+      include Courses::EditOptions::AgeRangeConcern
+
+      def level; end
+    end
+
+    klass.new
+  end
+
+  before do
+    allow(example_model).to receive(:level).and_return(level_value)
```

Comment:
I was surprised to see this here, feels like a weird way to do it. 

I feel an easier way to handle this would be to just make `level` an `attr_accessor` on the model, and in the before block do `example_model.level = level_value`, as this makes it clearer that it's essentially setting `level` on the model rather than stubbing something out

Reply:
Agreed. Particularly as in this case `example_model` is an anonymous class generated only for the purpose of testing.

Reply:
Has been resolved. Thanks for the feedback